pkg/apis/extensions/validation/validation_test.go

```diff
@@ -2326,33 +2326,6 @@ func TestValidateReplicaSet(t *testing.T) {
 }
 
 func TestValidatePodSecurityPolicy(t *testing.T) {
-	validPSP := func() *extensions.PodSecurityPolicy {
-		return &extensions.PodSecurityPolicy{
-			ObjectMeta: metav1.ObjectMeta{
-				Name:        "foo",
-				Annotations: map[string]string{},
-			},
-			Spec: extensions.PodSecurityPolicySpec{
-				SELinux: extensions.SELinuxStrategyOptions{
-					Rule: extensions.SELinuxStrategyRunAsAny,
-				},
-				RunAsUser: extensions.RunAsUserStrategyOptions{
-					Rule: extensions.RunAsUserStrategyRunAsAny,
-				},
-				FSGroup: extensions.FSGroupStrategyOptions{
-					Rule: extensions.FSGroupStrategyRunAsAny,
-				},
-				SupplementalGroups: extensions.SupplementalGroupsStrategyOptions{
-					Rule: extensions.SupplementalGroupsStrategyRunAsAny,
-				},
-				AllowedHostPaths: []extensions.AllowedHostPath{
```

Comment:
this is different between the two... I'd rather keep the example PSPs local to the tests where they are used

Reply:
This difference is irrelevant because test doesn't use `AllowedHostPaths`.

Reply:
I don't see how it could harm but I see that file has about 3k lines of code. I still think that such trivial duplication should be eliminated.

Reply:
making a test stub shared means you have to consider all tests that use it before modifying it. locality of reference for test stubs is preferred

Reply:
I agree but in many cases we don't care about pod properties we just need some valid pod. This is where this function is helpful. Copying&pasting it in many times has a downside that we need to update all the places when we'll add a new required field.